qa/pull-tester/rpc-tests.py

```diff
@@ -29,6 +29,7 @@
 import tempfile
 import re
 
+sys.path.append("qa/pull-tester/")
```

Comment:
Not sure how to fix this...

What about byte-compiling the rpc-tests.py instead of linking?